test/e2e/lib/hooks/browser/browser-logs.js

```diff
@@ -2,29 +2,17 @@
  * External dependencies
  */
 import fs from 'fs/promises';
+import path from 'path';
 import { logging } from 'selenium-webdriver';
 
-/**
- * Internal dependencies
- */
-import { generatePath } from '../../test-utils';
-
-export const saveBrowserLogs = async ( driver ) => {
-	try {
-		await Promise.allSettled(
-			[
-				[ () => driver.manage().logs().get( logging.Type.BROWSER ), 'console.log' ],
-				[ () => driver.manage().logs().get( logging.Type.PERFORMANCE ), 'performance.log' ],
-			].map( async ( [ logsPromise, file ] ) => {
-				const logs = await logsPromise();
-				return fs.writeFile( generatePath( file ), JSON.stringify( logs, null, 2 ) );
-			} )
-		);
-	} catch ( err ) {
```

Comment:
This function shouldn't assume that is ok to swallow the error, as it may be ok in Mocha but not in Jest (for example). 

The responsibility of that decision is for the hook adapter `test/e2e/lib/hooks/mocha.js`